src/index.js

```diff
@@ -153,7 +153,9 @@ export function configure(env) {
 			],
 
 			alias: {
-				preact: path.resolve(__dirname, 'preact.js'),
+				preact: path.resolve(__dirname, 'shims/preact.js'),
+				'preact-router': path.resolve(__dirname, 'shims/preact-router.js'),
+				'react-apollo': path.resolve(__dirname, 'shims/react-apollo.js'),
 				react: 'preact',
```

Comment:
Similar to your note on Darby's PR, should this be `react: 'preact-compat'`

Reply:
Fixed, and that allows this PR to obsolete https://github.com/Zimbra/zimlet-cli/pull/5.